tests/test_units/test_dataio.py

```diff
@@ -183,6 +181,107 @@ def test_deprecated_keys(globalconfig1, regsurf, key, value, expected_msg):
         edata.generate_metadata(regsurf, **kval)
 
 
+def test_access_ssdl_vs_classification_rep_include(globalconfig1, regsurf):
+    """
+    The access_ssdl is deprecated, and replaced by the 'classification' and
+    'rep_include' arguments. Test various combinations of these arguments.
+    ."""
+
+    # verify that a deprecation warning is given for access_ssdl argument
+    with pytest.warns(FutureWarning, match="'access_ssdl' argument is deprecated"):
+        exp = ExportData(
+            config=globalconfig1,
+            access_ssdl={"access_level": "restricted", "rep_include": True},
+        )
+        mymeta = exp.generate_metadata(regsurf)
+        assert mymeta["access"]["classification"] == "restricted"
+        assert mymeta["access"]["ssdl"]["rep_include"] is True
+
+    # 'access_ssdl' is not allowed together with any combination of
+    # 'classification' / 'rep_include' arguments
+    with pytest.raises(ValueError, match="Conflicting arguments"):
+        ExportData(
+            access_ssdl={"access_level": "restricted"}, classification="internal"
+        )
+    with pytest.raises(ValueError, match="Conflicting arguments"):
+        ExportData(access_ssdl={"rep_include": True}, rep_include=True)
+
+    with pytest.raises(ValueError, match="Conflicting arguments"):
+        ExportData(access_ssdl={"access_level": "restricted"}, rep_include=False)
```

Comment:
Are these conflicting...?

Reply:
OK I see the comment above, never allow `access_ssdl` together with the new ones 👍

Reply:
well maybe not the right word.. conflicting in mixing old and new_style.  I can try to reword 👍